examples/cc26xx/cc26xx-web-demo/coap-server.c

```diff
@@ -133,6 +134,7 @@ PROCESS_THREAD(coap_server_process, ev, data)
 
   rest_activate_resource(&res_batmon_temp, "sen/batmon/temp");
   rest_activate_resource(&res_batmon_volt, "sen/batmon/voltage");
+  rest_activate_resource(&res_adc_dio23, "adc/dio23");
```

Comment:
Should we follow the convention set by the batmon resources and go with sen/adc/dio23?